templates/clickhouse-server.service.j2

```diff
@@ -1,29 +0,0 @@
-{{ ansible_managed | comment }}
```

Comment:
For maintaining the same behavior till now, I would prefer to make the necessary changes than delete and use the default (with no option to customize) service file.

It's seems that we only had to update some lines from the service file like CapabilityBoundingSet and maybe the After and Wants lines like in here https://github.com/ClickHouse/ClickHouse/blob/b4b603e0b57bdd1ce373ca278b21a7267ffff2ee/packages/clickhouse-server.service